compiler/rustc_codegen_llvm/src/type_.rs

```diff
@@ -275,6 +279,9 @@ impl LayoutTypeMethods<'tcx> for CodegenCx<'ll, 'tcx> {
     fn cast_backend_type(&self, ty: &CastTarget) -> &'ll Type {
         ty.llvm_type(self)
     }
+    fn fn_decl_backend_type(&self, fn_abi: &FnAbi<'tcx, Ty<'tcx>>) -> &'ll Type {
+        fn_abi.llvm_type(self)
+    }
     fn fn_ptr_backend_type(&self, fn_abi: &FnAbi<'tcx, Ty<'tcx>>) -> &'ll Type {
         fn_abi.ptr_to_llvm_type(self)
     }
```

Comment:
I wonder if this method should have this default (or be inlined, but right now there's two callers):
```rust
self.type_ptr_to_ext(cx.data_layout().instruction_address_space)
```
(also `type_ptr_to_ext` should probably be renamed to `type_ptr_to_in_addrspace` or something)